app/assets/stylesheets/components/_base.sass

```diff
@@ -3,13 +3,15 @@
 @import 'button'
 @import 'container'
 @import 'event'
-@import 'form'
+@import 'form_body_layout'
```

Comment:
There too many layout components. They all do the same things almost. Try to merge this into a more generic component, agnostic of the content